Libraries/LibC/netdb.cpp

```diff
@@ -49,7 +48,18 @@ static hostent __gethostbyaddr_buffer;
 static char __gethostbyaddr_name_buffer[512];
 static in_addr_t* __gethostbyaddr_address_list_buffer[2];
 
-static auto services_file = Core::File::construct("/etc/services");
+static FILE* services_file = nullptr;
+const char* serv_file_loc = "/etc/services"; 
```

Comment:
Does this variable need to be accessed from outside of this file? If not, please make it `static`.
Also, it's very abbreviation heavy which we try to avoid. How about `services_path`?